### ISSUE TITLE
test: Create Postman collection

### ISSUE THREAD

**Issue by dricazenck** [Women-Coding-Community/wcc-backend#160]

**Comment by dricazenck** [comment]
@hannavas2022 I saw the status is in review, but I don't see PR's changes did you shared already?

**Comment by hannavas2022** [comment]
Hi @dricazenck! I wasn’t able to go through the instructions one more time to find out why I have a conflict with merging. I’m supposed to do it this evening.

**Comment by dricazenck** [comment]
I don't think so @ValeriyaMix, I am assigning to you and you can give a try. Here [is the PR ](https://github.com/Women-Coding-Community/wcc-backend/pull/165)@hannavas2022 started to work, but make sure you have the latest changes from our repository. 

You can take for reference [here](https://wcc-backend.fly.dev/swagger-ui/index.html) 
But I am not sure sure if has the latest version. You can start locally and you can see what are the latest APIs available to be documented. 

@nora-weisser what is the priority for this ticket? is something you still need?

**Comment by nora-weisser** [comment]
> I don't think so [@ValeriyaMix](https://github.com/ValeriyaMix), I am assigning to you and you can give a try. Here [is the PR ](https://github.com/Women-Coding-Community/wcc-backend/pull/165)[@hannavas2022](https://github.com/hannavas2022) started to work, but make sure you have the latest changes from our repository.
> 
> You can take for reference [here](https://wcc-backend.fly.dev/swagger-ui/index.html) But I am not sure sure if has the latest version. You can start locally and you can see what are the latest APIs available to be documented.
> 
> [@nora-weisser](https://github.com/nora-weisser) what is the priority for this ticket? is something you still need?

Hi @ValeriyaMix!
It would be great to work on this issue :)
